### ISSUE TITLE
Team page: information about team members

### ISSUE THREAD

**Issue by svitlanakalancha** [hpi-swt2/sport-portal#23]
As Team member I see information about other team members on the team page.

**Acceptance criteria:** 

- [x] Team page includes a table with following columns:
        -number
	-name of the person
	-panel with actions (send email, delete, assign team owner)

- [x] All new team members are automatically shown in the table

- [x] User can choose group of users with a checkbox and perform action on them (delete, email)
	
- [x] Checkboxes to choose several team members

**Out of scope:** 
Implementation of delete and email functions

<img width="372" alt="screen shot 2017-11-11 at 21 41 47" src="https://user-images.githubusercontent.com/27727717/32693310-2905d3f6-c729-11e7-9726-95c34593b688.png">

**Comment by bakoe** [comment]
depends on #27: *Roles in team*

**Comment by niconomaa** [comment]
### technical requirements of this issue:

> Team page includes a table with [...] columns:

- iterate over each team member
- insert action item and properties for each target member
- disable buttons depending on user's `Ability` to perform action on target member
    - check each action's `Ability` using CanCanCan`s `can?` method

> All new team members are automatically shown in the table

- load members upon sending `show` action of `TeamsController`

> User can choose group of users with a checkbox and perform action on them (delete, email) 
#### and
>  Checkboxes to choose several team members
- insert checkbox while iterating over members
- insert additional checkbox brought to youuuu by jQuery to (de)select all members at a time
- introduce new action in `TeamsController` to perform existing action functionality on the selected set of target members
   - extract existing actions' functionalities to be shared among perform on single and multiple 
   - rewrite `Ability` to make it work with block parameters only
   - explicitly reload team for each target member to avoid automatic model caching
   - catch members on which action could not be performed due to missing `Ability`
   - display target members on whom action could be performed to user

- write tests for all of this
